test/cpp/jit/test_gpu.cpp

```diff
@@ -18148,6 +18148,135 @@ TEST(NVFuserTest, FusionRfactorContigIDs_CUDA) {
   testValidate(&fusion, outputs, aten_inputs, {ref}, __LINE__, __FILE__);
 }
 
+TEST(NVFuserTest, FusionMMASwizzlePrimitiveFloatAcc_CUDA) {
+  FusionExecutor fe;
+
+  std::string kernel = R"(
+  __global__ void mma32x32x8_mini_gemm_float(Tensor<__half, 2> Ag, Tensor<__half, 2> Bg, Tensor<float, 2> Cg0, Tensor<float, 2> Cg1){
+    // Accumulator
+    float C[32];
+    // initialize C:
+    for(int i=0;i<32;i++){
+        C[i] = 0;
+    }
+    // Allocate Smem
+    __shared__ uint4 As_mem[32*8 / 8];
+    __shared__ uint4 Bs_mem[8*32 / 8];
+    auto As = reinterpret_cast<__half*>(As_mem);
+    auto Bs = reinterpret_cast<__half*>(Bs_mem);
+    // Compute lane_id
+    int lane_id = threadIdx.x % 32;
+    
+    // Mini prolog: (single warp)
+        // global load:
+    __half Ag_buffer[8];
+    *reinterpret_cast<uint4*>(Ag_buffer) = *reinterpret_cast<uint4*>(&Ag[threadIdx.x*8]);
+    __half Bg_buffer[8];
+    *reinterpret_cast<uint4*>(Bg_buffer) = *reinterpret_cast<uint4*>(&Bg[threadIdx.x*8]);
+        // Write into smem: (should consider cleaner utility for vectorization)
+    *reinterpret_cast<uint4*>(
+        &As[
+                // Utility to compose a sequence of swizzles
+            mem_swizzle::swizzle_sequence<
```

Comment:
This would be useful when we come to the bank conflict removal step. My current understanding now is that we have swizzle for mma and swizzle for bank conflict (will be in the next few PRs). The former is a function of the instruction we are using and the latter is a function of CTA tile size and vectorization. So swizzle sequence is an interface for fuser to configure them separately and the kernel will apply them sequentially.

Reply:
So it's just a mechanism to go from one swizzle format to another?

Reply:
That could be one way to use this utility as well. 

The scenario I was thinking was that the swizzled format in smem in gemm is always a composition of 2 because we want to satisfy mma requirement and avoid bank conflict at the same time. 

if say we have a smem dimension of 128x32 f16, with 128b vectorization, and storing a mma operandA, then we need a swizzle that satisfies mma requirement and avoids bank conflict when we load a column. 

This could be done with:
`cyclic shift blocks of 128b by 8, every 2 rows`(in a separate PR) and then `mma operand A swizzle`.

So `swizzle sequence` was used as a utility to configure these 2 separately and generate the composed format.